Earth_Radius: 1
Jupiter_Radius: 5.203
Mars_Radius: 1.524
Mercury_Radius: 0.387
Neptune_Radius: 30.06
Saturn_Radius: 9.539
Titan_Radius: 0.3
Uranus_Radius: 19.19
Venus_Radius: 0.723
